File: 4th Semester/Design and Programming Environments/labs/3rd lab/C#/basket/domain/Purchase.cs
namespace basket.domain;

public class Purchase:Entity<long>
{
    public string client;
    public long game;
    public int seats;
    public string address;

    public Purchase(string client, long game, int seats, string address)
    {
        this.client = client;
        this.game = game;
        this.seats = seats;
        this.address = address;
    }

    public string get_client()
    {
        return client;
    }

    public void set_client(string client)
    {
        this.client = client;
    }

    public long get_game()
    {
        return game;
    }

    public void set_game(long game)
    {
        this.game = game;
    }

    public int get_seats()
    {
        return seats;
    }

    public void set_seats(int seats)
    {
        this.seats = seats;
    }

    public string get_address()
    {
        return address;
    }

    public void set_address(string address)
    {
        this.address = address;
    }
}

File: 4th Semester/Design and Programming Environments/labs/5th lab/Java/BasketModel/Purchase.java
package ro.mpp2024;

import java.io.Serializable;

public class Purchase extends Entity<Long> implements Serializable {
    private String client;
    private Long game;
    private int seats;
    private String address;
    public Purchase(Long id, String client, Long game, int seats, String address) {
        super.set_identitykey(id);
        this.client = client;
        this.game = game;
        this.seats = seats;
        this.address = address;
    }
    public Long get_identitykey(){
        return super.get_identitykey();
    }
    public void set_identitykey(Long id){
        super.set_identitykey(id);
    }
    public String getClient() {
        return client;
    }

    public void setClient(String client) {
        this.client = client;
    }

    public Long getGame() {
        return game;
    }

    public void setGame(Long game) {
        this.game = game;
    }

    public int getSeats() {
        return seats;
    }

    public void setSeats(int seats) {
        this.seats = seats;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }
}
